docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode
+
+## Install toolchains
+
+First, you need ONE toolchains. Let's install ONE toolchains. (You should install the in-house backend to follow this.)
+
+![toolchain](https://user-images.githubusercontent.com/7223627/172834540-945ed5f9-82b0-4388-bd46-4ea10587d701.gif)
```

Comment:
```suggestion
![install](https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif)
<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500/>
```

<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500/>

/cc @hyunsik-yoon @YongseopKim What about this gif file?

Reply:
This looks so much better!